umbra-js/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "composite": true,
     "declarationMap": true,
+    "skipLibCheck": true,
     "outDir": "build",
     "resolveJsonModule": true,
```

Comment:
(arbitrary comment)

The logo is now messed up. Do we have a svg version of the logo.

Reply:
It seems like the width is set to 56px. If we had an svg or a 56x56 image of the logo I think the issue would be resolved.

Reply:
Which logo, the Umbra logo? If so I don't think we have one, I made the logo in powerpoint a few years ago and don't think I even have the original files anymore 😅 Might be able to import it into some image editing software and have it generate an SVG from it, but not sure